cmsdb/processes/hh.py

```diff
@@ -133,106 +138,96 @@
-    xsecs={
-        13: Number(0.1),  # TODO
-        13.6: Number(0.1),  # TODO
-    },
     aux={"production_mode_parent": hh_ggf},
 )
 
 hh_ggf_kl1_kt1_c23 = hh_ggf.add_process(
     name="hh_ggf_kl1_kt1_c23",
     id=21008,
-    xsecs={
-        13: Number(0.1),  # TODO
-        13.6: Number(0.1),  # TODO
-    },
     aux={"production_mode_parent": hh_ggf},
 )
 
 hh_ggf_kl1_kt1_c2m2 = hh_ggf.add_process(
     name="hh_ggf_kl1_kt1_c2m2",
     id=21009,
-    xsecs={
-        13: Number(0.1),  # TODO
-        13.6: Number(0.1),  # TODO
-    },
     aux={"production_mode_parent": hh_ggf},
 )
 
-# Source: xsecs were given in fb but are converted to pb,
-# uncertainties in %, values are taken for m_H = 125 GeV
-# https://twiki.cern.ch/twiki/bin/view/LHCPhysics/LHCHWGHH?rev=90#HHjj_VBF
+"""
+Source: https://twiki.cern.ch/twiki/bin/view/LHCPhysics/LHCHWGHH?rev=98#HHjj_VBF
+Extraction workflow for VBF cross sections:
+- values are taken for m_H = 125 GeV
+- LO values from XSDB, see e.g.:
+https://xsdb-temp.app.cern.ch/xsdb/?columns=67108863&currentPage=0&pageSize=10&searchQuery=process_name%3D%5EVBFHHto2B2Tau_CV.%2B_C2V.%2B_C3.%2B_.%2A13p6TeV.%2A%24  # noqa
+- xsecs were given in fb but are converted to pb
+- k-factor derived by comparing SM LO value to N3LO value from Twiki above
+- it is assumed to be constant across all other kappas and multiplied to corresponding LO values
+- uncertainties are constant and taken from the Twiki above
+"""
+
+hh_vbf_uncs_13p0 = {
+    "scale": (0.0005j, 0.0004j),
+    "pdf": 0.027j,
+}
+hh_vbf_uncs_13p6 = {
+    "scale": (0.0005j, 0.0003j),
+    "pdf": 0.027j,
+}
 
 hh_vbf = hh.add_process(
     name="hh_vbf",
     id=22000,
     label=r"$HH_{vbf}$",
-    xsecs={
-        13: 0.001 * Number(1.726, {
-            "scale": (0.0003j, 0.0004j),
-            "pdf": 0.021j,
-        }),
-        13.6: Number(0.1),  # TODO
-    },
     aux={"production_mode_parent": hh},
 )
 
 hh_vbf_kv1_k2v1_kl1 = hh_vbf.add_process(
     name="hh_vbf_kv1_k2v1_kl1",
     id=22001,
     xsecs={
-        13: Number(0.0017260, {
-            "scale": (0.0003j, 0.0004j),
-            "pdf": 0.021j,
-        }),
-        13.6: Number(0.1),  # TODO
+        13: Number(1.687, hh_vbf_uncs_13p0) * 0.001,
+        13.6: Number(1.874, hh_vbf_uncs_13p6) * 0.001,
     },
     aux={"production_mode_parent": hh_vbf},
 )
 
+hh_vbf_k_13p0 = hh_vbf_kv1_k2v1_kl1.get_xsec(13).nominal / 1.626
+hh_vbf_k_13p6 = hh_vbf_kv1_k2v1_kl1.get_xsec(13.6).nominal / 1.912
+
 hh_vbf_kv1_k2v1_kl0 = hh_vbf.add_process(
     name="hh_vbf_kv1_k2v1_kl0",
     id=22002,
     xsecs={
-        13: Number(0.0046089, {
-            "scale": (0.0003j, 0.0004j),
-            "pdf": 0.021j,
-        }),
-        13.6: Number(0.1),  # TODO
+        13: Number(4.337 * hh_vbf_k_13p0, hh_vbf_uncs_13p0) * 0.001,
+        # no 13p6TeV sample generated, but we could use the scaling formula in the hh tools
+        # 13.6: Number(TODO * hh_vbf_k_13p6, hh_vbf_uncs_13p6) * 0.001,
```

Comment:
maybe we can also use the same k-factor for 13p6 as for 13p0 (I think I checked this for ggHH at some point and there the k-factors were basically identical)

Reply:
They aren't. Both the NLO samples and the NNLO cross sections changed in the meantime.